scylla/src/transport/locator/mod.rs

```diff
@@ -3,10 +3,11 @@ pub mod replicas;
 pub mod replication_info;
 #[cfg(test)]
 mod test;
-pub mod token_ring;
+pub(crate) mod token_ring;
 
 pub use precomputed_replicas::PrecomputedReplicas;
-pub use replicas::{NodeRef, Replicas, EMPTY_REPLICAS};
+pub use replicas::NodeRef;
+pub(crate) use replicas::{Replicas, EMPTY_REPLICAS};
```

Comment:
I don't think that anything except `TokenRing` and `NodeRef` (this 2 types are part of `locator::ReplicaLocator` public API) should be re-exported.